test/new-e2e/tests/agent-subcommands/check/parse.go

```diff
@@ -13,33 +13,37 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-type root struct {
-	Aggregator aggregator `json:"aggregator"`
+// Root is the root object of the check command output
```

Comment:
Would it make sense to move this code to pkg? I don't think creating dependencies between tests are a good thing.

Reply:
If we do that we have to make a new module so that it can be imported...
I could create a shared package somewhere in `test/new-e2e` though 🤔

Reply:
I meant `test/new-e2e/pkg/`

Reply:
I'm going to do that 👍